auth-api/src/auth_api/resources/org.py

```diff
@@ -195,6 +195,58 @@ def get(org_id):
         return response, status
 
 
+@cors_preflight('GET,POST,PUT,OPTIONS')
+@API.route('/<string:org_id>/login_options', methods=['GET', 'POST', 'PUT', 'OPTIONS'])
```

Comment:
We need a techdebt to decide on the rest endpoint resource naming. Pay uses hyphons.

Reply:
I will change this now..better is hyphen